### PR TITLE
Burak-pm-archive-info

### PR DESCRIPTION
**Pull request by myniqx**
https://github.com/freedomcombination/monorepo/issues/1215

Ekran görüntüsü 2024-04-18 135539

i added this screenshot because there is no post with archiveId in dev. this is how it looks like.

and:
image
this is how it looks like when there is no archiveId provided. and those stat section appears only if you have roles.

### CODE REVIEW

File: packages/ui/src/post-maker/PostMakerTweetCard/PostSentenceRefDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+  const archiveContent = data?.data
+
+  if (sentence == null) return null
+
+  const loadingInfo = () => {
+    return (
+      <Center background={'white'} borderRadius={'lg'} p={6}>
+        {isLoading ? (
+          <Text>Archive is loading...</Text>
```
Comment (7alip) [inline packages/ui/src/post-maker/PostMakerTweetCard/PostSentenceRefDrawer.tsx:34]: Can be a <Spinner /> or <SkeletonText /> component

File: packages/ui/src/post-maker/PostMakerTweetCard/PostSentenceRefDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        ) : sentence.archiveId ? (
+          <Text>Archive with id {sentence.archiveId} not found.</Text>
```
Comment (7alip) [inline packages/ui/src/post-maker/PostMakerTweetCard/PostSentenceRefDrawer.tsx:36]: This message could be a general message, like: "No details found for this post" in an Alert component.

File: packages/ui/src/post-maker/PostMakerTweetCard/PostSentenceRefDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        ) : (
+          <Text>No archive id provided.</Text>
```
Comment (7alip) [inline packages/ui/src/post-maker/PostMakerTweetCard/PostSentenceRefDrawer.tsx:38]: This doesn't mean anything to public users.

File: packages/ui/src/post-maker/PostMakerTweetCard/PostSentenceReference.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  const { sentence } = usePostContext()
+  const { setSentence } = useHashtagContext()
+
+  return (
+    <IconButton
+      aria-label="Manage sentences"
+      icon={<FaInfo />}
+      variant={'outline'}
+      isRound
+      onClick={() => setSentence(sentence)}
```
Comment (7alip) [inline packages/ui/src/post-maker/PostMakerTweetCard/PostSentenceReference.tsx:19]: Do we use this to display the sentence inside the drawer? If so, it seems unnecessary to me.

**Review by 7alip — COMMENTED** [review]
Thanks

**Comment by myniqx** [comment]
image

**Review by 7alip — APPROVED** [review]
Added translations, removed stats as the count is already on the card. Updated spacings

**Pull request merged**